src/functionalTest/java/uk/gov/hmcts/dts/fact/CourtsEndpointTest.java

```diff
@@ -192,4 +194,20 @@ public void shouldNotReturnDuplicatesForCourtsWithMultipleAddresses() {
         final List<CourtReference> courts = Arrays.asList(response.getBody().as(CourtReference[].class));
         assertThat(courts.size()).isEqualTo(1);
     }
+
+    @Test
+    public void shouldReturnCourtsByCourtTypes() {
+        final var response = doGetRequest(COURT_SEARCH_BY_COURT_TYPES_ENDPOINT + "tribunal,family");
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+
+        final List<Court> courts = Arrays.asList(response.getBody().as(Court[].class));
+        assertThat(courts.get(0).getCourtTypes()).anyMatch(type -> type.contains("Tribunal") || type.contains("Family Court"));
```

Comment:
anyMatch isn't an assertion for true/false right? Just returns a true or a false if any matches

I think you want assertTrue here? 

Might be easiest to do a stream and assertTrue and then do all match type (either tribunal or family) 

also family court as mentioned here would be family right?

Reply:
changed to  assertTrue(courts.get(0).getCourtTypes()
                       .stream()
                       .anyMatch(type -> type.contains("Tribunal") || type.contains("Family Court")));

the returning court types will have the name returned not the search column like  "types": [
            "Family Court",
            "County Court"
        ], 
so that's why we checking for Family Court